.github/workflows/reusable-end-to-end-testing.yml

```diff
@@ -7,63 +7,14 @@ on:
         required: true
         type: string
         description: 'Image tag to build and push.'
-      registry_name:
-        required: false
-        type: string
-        description: 'Registry name used for container image names. Default is `ghcr.io/openclarity`.'
-        default: ghcr.io/openclarity
```

Comment:
we could keep this field to use in `Run end to end tests` step

Reply:
As far as I know we only run e2e tests as part of the CI run, I deleted it because ghcr will never be used on CI runs.

Reply:
Added back this field! 👍